### ISSUE TITLE
anti_sample fails 

### ISSUE THREAD

**Issue by MarkEdmondson1234** [MarkEdmondson1234/googleAnalyticsR#32]
Example account with 91.98% sampling per day.

**Comment by MarkEdmondson1234** [comment]
It will fail if using a heavily filtered profile (can we detect that?)
A retry for fetches that are sampled?

**Comment by MarkEdmondson1234** [comment]
```
Downloaded [22967] rows from a total of [23022].
Error in seq.default(from = 50000, to = all_rows, by = reqRowLimit) : 
  wrong sign in 'by' argument
```

```
anti_sample set to TRUE. Mitigating sampling via multiple API calls.
Finding how much sampling in data request...
Downloaded [10] rows from a total of [183079].
Data is sampled, based on 54.7% of sessions.
Error in seq.default(date_range[1], date_range[2], by = 1) : 
  'from' cannot be NA, NaN or infinite
```

**Comment by MarkEdmondson1234** [comment]
It will fail for big accounts if you include today's date, as that data is not `golden` the row limit is incorrect in the API.

**Comment by MarkEdmondson1234** [comment]
Need to put the dates in as `Date` objects currently to stop these errors:
```
Error in seq.default(from = 50000, to = all_rows, by = reqRowLimit) : 
  wrong sign in 'by' argument
```

Change to check for that.

**Comment by michaelmb** [comment]
I ran into the date error issue, used as.Date() and was able to pull the report without issue after. Thanks for noting!

**Comment by MarkEdmondson1234** [comment]
Cool, it should be fixed in the version on github, CRAN from 0.4.0
